ms2deepscore/BinnedSpectrum.py

```diff
@@ -0,0 +1,39 @@
+class BinnedSpectrum:
+    """Binned spectrum for use with MS2DeepScore."""
+    def __init__(self, binned_peaks: dict, metadata: dict):
+        self.peaks = binned_peaks
```

Comment:
```suggestion
    def __init__(self, binned_peaks: dict, metadata: dict):
        self.binned_peaks = binned_peaks
```

Reply:
Or the other way around (using `peaks` for both)

Reply:
Good point. I'll change it elsewhere, too.